v0/destinations/lambda/transform.js

```diff
@@ -0,0 +1,91 @@
+const _ = require("lodash");
+const {
+  getErrorRespEvents,
+  getSuccessRespEvents,
+  CustomError
+} = require("../../util");
+
+const DEFAULT_INVOCATION_TYPE = "Event";
+const MAX_PAYLOAD_SIZE_IN_KB = 256;
+
+// Returns a transformed payload, after necessary property/field mappings.
+function process(event) {
+  if (!event.destination.Config) {
+    throw new CustomError("destination.Config cannot be undefined", 400);
+  }
+  return {
+    payload: JSON.stringify(event.message),
+    destConfig: event.destination.Config
+  };
+}
+
+// Returns an array of payloads within the size limit
+function payloadSizeRegulator(payload) {
```

Comment:
we better ask max_batch_size from user while configuring the Lambda function as we are not aware about what exact processing they are doing in their Lambda function and if we try to pack max possible events based on Payload limits then lambda might timeout in processing them.

Reply:
Added `maxBatchSize` input and using it while batching